File: Day 8 : Dictionaries and Maps/Solution.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
if __name__ == '__main__' :
    no_of_entries = int(input())
    phonebook = {}
    for i in range(no_of_entries):
        details = input()
        details = details.split()
        phonebook[details[0]] = details[1]
    while True:
        try:
            search = input()
            got = 0
            if search in phonebook:
                print(search + '=' + phonebook[search])
            else:
                print("Not found")
        except:
            break

File: Day 8 : Dictionaries and Maps/Solution.js
function processData(input) {
    inputList = input.split("\n");
    
    const phonebook = {};
    for(let i = 1; i < parseInt(inputList[0]) + 1; i++){
        let entry = inputList[i].split(" ");
        phonebook[entry[0]] = entry[1];
    }
    
    for(let i = parseInt(inputList[0]) + 1; i < inputList.length; i++) {
        if(phonebook[inputList[i]]){
            console.log(inputList[i] + "=" + phonebook[inputList[i]]);
        } else {
            console.log("Not found");
        }
    }
}
